container-build/sources.list

```diff
@@ -0,0 +1 @@
+deb [arch=amd64] https://download.docker.com/linux/debian stretch stable
```

Comment:
Why is the "[arch=amd64]" needed? How does it work when doing an arm64 build?

Reply:
good catch. that specifier shouldn't be there. the arm64 build works because this sources.list is only for installing docker-ce-cli in the top level container used to run `make`.

Reply:
fixed in 5c91fcd